core/server/api_container/server/startosis_engine/runtime_value_store/recipe_result_repository.go

```diff
@@ -143,7 +143,12 @@ func (repository *recipeResultRepository) Get(
 			var comparableValue starlark.Comparable
 			comparableInt, err := strconv.Atoi(stringifiedComparable)
 			if err != nil {
-				comparableValue = starlark.String(stringifiedComparable)
```

Comment:
Can we reduce the nesting here?

Creating a function returning `Comparable, error` with the following structure:

```
comparableInt, err := strconv.Atoi(stringifiedComparable)
if err != nil {
    return comparableInt
}
comparableBool, err := strconv.ParseBool(stringifiedComparable)
if err != nil {
    return comparableBool
}
...
```